test/linalg/svd.jl

```diff
@@ -4,6 +4,31 @@ using Base.Test
 
 using Base.LinAlg: BlasComplex, BlasFloat, BlasReal, QRPivoted
 
+@testset "Simple svdvals / svdfact tests" begin
+    ≈(x,y) = isapprox(x,y,rtol=1e-15)
```

Comment:
I admit this is a bit provocative :-)

On the plane trip, we were trying to think of ways we could have a "test context" where you have a tighter given tolerance than the default loose `rtol`.  The problem I have with using `@test ... rtol=epsilon` each time is that it's verbose and ugly - I'd really like this to be set as a default at the `@testset` level.

So the idea here is as a way of defaulting `rtol`.  I'm not tied to this specific solution, and I'm happy to change it for this case.  But I would like a nicer way to deal with this plus some other testset related issues.  Would be great to chat to some people who care about the test infrastructure while I'm at juliacon - any suggestions?